Sources/Core/Extensions/String+Extensions.swift

```diff
@@ -18,20 +18,33 @@
 
 import Foundation
 
-// This will eventually be replaced by the rewrite of FBSDKAccessTokenCache
-// for now it is needed as a transient dependency of AccessTokenWallet (via Settings)
+extension String {
```

Comment:
Somewhat wary of extending base types but this is common enough that it doesn't seem likely to burn us.

Reply:
If we keep this internal, this shouldn't be an issue.